Assets/MixedRealityToolkit.SDK/Features/Input/Handlers/TransformScaleHandler.cs

```diff
@@ -0,0 +1,175 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using UnityEngine;
+
+namespace Microsoft.MixedReality.Toolkit.UI
+{
+    /// <summary>
+    /// Helper class containing move/rotate/scale related utility functions.
```

Comment:
Please adjust this comment. Its not a helper class anymore. We might want to add this component to the docs of manipulation handler or bounding box as well. And make sure its used in one of the example scenes so ppl see how to use it there as well